pkg/manifest/os.go

```diff
@@ -791,6 +793,21 @@ func (p *OS) serialize() osbuild.Pipeline {
 		}))
 	}
 
+	if len(p.CACerts) > 0 {
+		for _, cc := range p.CACerts {
+			files, err := osbuild.NewCAFileNodes(cc)
+			if err != nil {
+				panic(err.Error())
```

Comment:
(super nitpick) `panic(err)` would also work (and is slightly shorter)

Reply:
This looks like a pattern in the whole file, I would rather not change it in this PR.